app/assets/stylesheets/analytics/analytics.scss

```diff
@@ -0,0 +1,79 @@
+@import url(https://fonts.googleapis.com/css?family=Nunito:300,400);
+
+$accent: #30385f;
+$dark: #E3E3E3;
+$medium: #EBEBEB;
+$light: #F5F5F5;
+
+.style {
+  font-family: 'Nunito', sans-serif;
+  font-weight: 200;
+}
+//*************************** CHARTS ***************************************
+
+.chart {
+  width: 100%;
+}
+
+.stacked-bar {
+  height: 767px;
+}
+
+.pie {
+  height: 367px;
+}
+
+.timeline-chart {
+  height: 288px;
+}
+
+// ************************ GRID FORMATTING *********************************
+
+.fit {
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-bottom: 0;
+}
+
+.dark {
+  background-color: $dark;
+}
+
+.medium {
+  background-color: $medium;
+}
+
+.light {
+  background-color: $light;
+}
+
+// ************************ TEXT FORMATTING *********************************
+
+
+.heading {
+  color: black;
+}
+
+.fancy {
+  margin: 0;
+  line-height: 4em;
+  font-weight: 200;
+  font-size: 1.13em;
+  text-transform: uppercase;
+
+  &.header {
+    padding-left: 15px;
```

Comment:
Properties should be ordered background-color, padding-left, position, width, z-index